package/MDAnalysis/coordinates/null.py

```diff
@@ -60,4 +60,3 @@ def _write_next_frame(self, obj):
         except AttributeError:
             errmsg = "Input obj is neither an AtomGroup or Universe"
             raise TypeError(errmsg) from None
-        pass
```

Comment:
Doesn't seem like this was required anyways?

Reply:
yep